corehq/apps/analytics/templates/analytics/initial/gtm.html

```diff
@@ -5,8 +5,8 @@
   {% initial_analytics_data 'gtm.userIsDimagi' request.couch_user.is_dimagi %}
   {% initial_analytics_data 'gtm.userIsCommCareUser' request.couch_user.is_commcare_user %}
 {% endif %}
-{% if domain %}
-  {% initial_analytics_data 'gtm.domain' domain %}
+{% if request.domain %}
+  {% initial_analytics_data 'gtm.domain' request.domain %}
```

Comment:
What about this? Not sure if prioritizing `domain` over `request.domain` is best, but seems to preserve existing behavior the best.
```suggestion
{% if domain or request.domain %}
  {% initial_analytics_data 'gtm.domain' domain or request.domain %}
```

Reply:
That's fair. I don't think there's any circumstance where we have domain data and would want to exclude it, so this is probably the best short-term fix, but is probably long-term confusing

Reply:
e91fe3b90506a8edfb1e9c693e21c115514b7f72